### DISCUSSION TITLE
Contribution timeline bug - repo name cut off

### DISCUSSION THREAD

**Discussion by sryze** [community/community#17426] [Other Feature Feedback, Questions, & Ideas]
I notice when I open my profile and scroll down to the list of repositories contributed recently, the names of those repos are always cut off, even though there is plenty of space left for them to be displayed in full. 

![github-timeline-bug](https://user-images.githubusercontent.com/4525736/170817675-6e093873-2fa3-45af-95dc-7eea005296e8.png)

This bug makes it a little difficult to view someone's contribution timeline.

Browser: Firefox 100.0.2 on Windows

**Comment by mohamed19871908** [comment]
‫في السبت، 28 مايو 2022 في 11:33 ص تمت كتابة ما يلي بواسطة ‪Sergey
Zolotarev‬‏ ***@***.***‬‏>:‬
